app/views/steps/conviction/known_conviction_date/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<% title t('.page_title') %>
+
+<div class="govuk-width-container">
+  <%= step_header %>
+
+  <main class="govuk-main-wrapper">
```

Comment:
Could we add `id="main-content"` onto the `<main>` tag please :)? This is so that the skip link will work

Reply:
I have add the `id="main-content"` onto the `<main>` tag.  Also updated the template to add the id when we generate new edit pages.